docs/getting_started/install_ibexa_dxp.md

```diff
@@ -114,58 +114,32 @@ This operation is performed only once, when you install [[= product_name =]] for
 
 To use Composer to instantly create a project in the current folder with all the dependencies, run the following command:
 
-!!! note "Using PHP 8.3 (recommended)"
+=== "[[= product_name_headless =]]"
 
-    === "[[= product_name_headless =]]"
-
-        ``` bash
-        composer create-project ibexa/headless-skeleton .
-        ```
-
-    === "[[= product_name_exp =]]"
-
-        ``` bash
-        composer create-project ibexa/experience-skeleton .
-        ```
-
-    === "[[= product_name_com =]]"
-
-        ``` bash
-        composer create-project ibexa/commerce-skeleton .
-        ```
-
-??? note "Using PHP 8.2 or older"
-
-    If you're using PHP 8.2 or any older version, use a different set of commands:
-
-    === "[[= product_name_headless =]]"
+    ``` bash
+    composer create-project ibexa/headless-skeleton .
+    ```
 
-        ``` bash
-        composer create-project ibexa/headless-skeleton --no-install .
-        composer update
-        ```
+=== "[[= product_name_exp =]]"
 
-    === "[[= product_name_exp =]]"
+    ``` bash
+    composer create-project ibexa/experience-skeleton .
+    ```
 
-        ``` bash
-        composer create-project ibexa/experience-skeleton --no-install .
-        composer update
-        ```
+=== "[[= product_name_com =]]"
 
-    === "[[= product_name_com =]]"
+    ``` bash
+    composer create-project ibexa/commerce-skeleton .
+    ```
 
-        ``` bash
-        composer create-project ibexa/commerce-skeleton --no-install .
-        composer update
-        ```
 
 !!! tip "Authentication token"
 
     <a id="authentication-token"></a>If you added credentials to the `COMPOSER_AUTH` variable, at this point add this variable to `auth.json` (for example, by running `echo $COMPOSER_AUTH > auth.json`).
 
 !!! tip
 
-    You can set [different version constraints](https://getcomposer.org/doc/articles/versions.md), for example, specific tag (`[[= latest_tag_4_6 =]]`), version range (`~4.6.10`), or stability (`^4.6@rc`):
+    You can set [different version constraints](https://getcomposer.org/doc/articles/versions.md), for example, specific tag (`[[= latest_tag_5_0 =]]`), version range (`~5.0.1`), or stability (`^5.0@rc`):
 
     ``` bash
     composer create-project ibexa/experience-skeleton:[[= latest_tag_4_6 =]] .
```

Comment:
Maybe use the new `latest_tag_5_0` you added to mkdocs.yaml:
```suggestion
    composer create-project ibexa/experience-skeleton:[[= latest_tag_5_0 =]] .
```

Reply:
Definitely, nice catch!